apps/communications/contacts/js/views/details.js

```diff
@@ -624,6 +631,36 @@ contacts.Details = (function() {
     cover.dataset.imgHash = '';
   };
 
+  var shareContact = function cd_shareContact() {
+    if (fb.isFbContact(contactData)) {
+      alert(_('facebook-export-forbidden'));
```

Comment:
Actually, we shouldn't need to recheck isnt? This is enabled just for non fb contacts